firebase-functions/functions/src/choiceOperations.ts

```diff
@@ -0,0 +1,32 @@
+﻿import { Player } from './models/player';
+
+export function hasEveryoneChosen(players: Array<FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>>): boolean {
+  return players.every(player => (player.data() as Player).choice);
+}
+
+export function calculateLosers(players: Array<FirebaseFirestore.QueryDocumentSnapshot<FirebaseFirestore.DocumentData>>): string[] {
```

Comment:
Sorry, I am confused, `calculateUsers` means eleminate them?

Reply:
It's calculate LOSERS :)
Line 18 tells you what I plan to do with those. Is it clearer now?

Reply:
I have a problem with the word "calculate" :)

Reply:
Yeah it is awkward. How about the evergreen `getLosers` then?

Reply:
Done